app/controllers/reminders_controller.rb

```diff
@@ -1,9 +1,13 @@
 class RemindersController < ApplicationController
+  before_action :authenticate_user!
   before_action :set_reminder, only: %i[ show edit update destroy ]
 
   # GET /reminders or /reminders.json
   def index
-    @reminders = Reminder.all
+    @subscriptions = current_user.subscriptions
+    @subscriptions.each do |subscription|
+      @reminders = subscription.reminders.all
```

Comment:
did you mean to assign `@reminders` to subscription.reminders.all or did you mean to append? If the latter, you can just do 
`@reminders = Reminder.where(user: current_user)`